index.html

```diff
@@ -300,10 +350,60 @@ <h2>
             </dt>
             <dd>
               <p>
-                Assumes values from the <code><a>CursorCaptureConstraint</a></code> enumeration that 
+                Assumes values from the <code><a>CursorCaptureConstraint</a></code> enumeration that
                 determines if and when the cursor is included in the captured display surface.
               </p>
-            </dd> 
+            </dd>
+          </dl>
+        </section>
+        <section>
+          <h2>
+            Extensions to <a class="idlType">MediaTrackSettings</a>
+          </h2>
+          <p>
+            When the <a href="https://www.w3.org/TR/mediacapture-streams/#constrainable-interface">getSettings()</a>
+            method is invoked on a video stream track, the user agent must return the extended
+            <a href="https://www.w3.org/TR/mediacapture-streams/#media-track-settings">MediaTrackSettings</a>
+            dictionary, representing the current status of the underlying user agent.
+          </p>
+          <pre class="idl">partial dictionary MediaTrackSettings {
+             DOMString displaySurface;
+             boolean   logicalSurface;
+             DOMString   cursor;
+};</pre>
+          <dl data-link-for="MediaTrackSettings" data-dfn-for="MediaTrackSettings" class=
+          "dictionary-members">
+            <dt>
+              <dfn><code>displaySurface</code></dfn> of type <span class=
+              "idlMemberType"><a>DOMString</a></span>
+            </dt>
+            <dd>
+              <p>
+                The type of <a>display surface</a> that is being captured. This assumes values from
+                the <code><a>DisplayCaptureSurfaceType</a></code> enumeration.
+              </p>
+            </dd>
+            <dt>
+              <dfn><code>logicalSurface</code></dfn> of type <span class=
+              "idlMemberType"><a>boolean</a></span>
+            </dt>
+            <dd>
+              <p>
+                A value of <code>true</code> indicates capture of a <a>logical display surface</a>;
+                a value of <code>false</code> indicates a capture capture of a <a>visible display
+                surface</a>.
+              </p>
+            </dd>
+            <dt>
+               <dfn><code>cursor</code></dfn> of type <span class=
+               "idlMemberType"><a>DOMString</a></span>
+            </dt>
+            <dd>
+              <p>
+                Assumes values from the <code><a>CursorCaptureConstraint</a></code> enumeration that
+                determines if and when the cursor is included in the captured display surface.
+              </p>
+            </dd>
```

Comment:
I believe we also need extensions to [MediaTrackCapabilities](https://w3c.github.io/mediacapture-main/getusermedia.html#dom-mediatrackcapabilities) for this to be complete.

Reply:
> I didn't add an extension on MediaTrackCapabilities as ranges don't really make sense for these properties.

String-based and boolean constrainable properties have "ranges" in the form of arrays containing all valid values.